cmd/terramate/cli/cloud.go

```diff
@@ -254,6 +259,73 @@ func (c *cli) cloudSyncAfter(run runContext, res runResult, err error) {
 	if run.CloudSyncDriftStatus {
 		c.cloudSyncDriftStatus(run, res, err)
 	}
+
+	if run.CloudSyncPreview {
+		c.doPreviewAfter(run, res)
+	}
+}
+
+func (c *cli) doPreviewBefore(run runContext) {
+	stackPreviewID := c.cloud.run.stackPreviews[run.Stack.ID]
+	if err := c.cloud.client.UpdateStackPreview(time.Second*60,
```

Comment:
It's better if the sdk methods receive a `ctx` instead of arbitrary timeouts.
By passing a timeout we lose control. A context managed in the `cli` package will be even more important with `--parallel` because then we can `cancel()` all pending operations on first failed stack (or on user's CTRL-C).

also, check the timeouts here: https://github.com/terramate-io/terramate/blob/v0.4.5/cmd/terramate/cli/cloud.go#L31-L35

if a timeout is needed, use the this value, so we have a single place to update.